docs/topics/readme/cd-rest-http-thorntail.properties

```diff
@@ -0,0 +1,8 @@
+localRunCMD=mvn wildfly-swarm:run
```

Comment:
The plugin is different for Thorntail. I believe it is now `thorntail:run` or similar @Ladicek or @kenfinnigan can confirm

Reply:
Correct, `mvn thorntail:run`.